### PR TITLE
Efp 17 user change password

### PR DESCRIPTION
**Pull request by typer177**
[EFP-17](https://codelation.atlassian.net/browse/EFP-17)

### CODE REVIEW

**Review by t-whitney — CHANGES_REQUESTED** [review]
- If I add an email that doesn't exist in the forgot password email field, I'm taken back to the login page without any indication of what happened. It would be useful to have an error message in this case.
- There should also be some sort of instructions for successfully sending the reset email. Right now it's really unclear what to do once you click reset (unless you get an email notification, but I don't think we should rely on that)

File: src/components/ForgotPage/ResetPage.js [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    };
+
+    handleInputChangeFor = (propertyName) => (event) => {
+        this.setState({
+         [propertyName]: event.target.value
+        });
+    };
+
+    render() {
+        console.log(this.props)
```
[the remark this answers is not in the document]
Comment (typer177) [reply src/components/ForgotPage/ResetPage.js:60]: cleaned up

**Comment by typer177** [comment]
>     * If I add an email that doesn't exist in the forgot password email field, I'm taken back to the login page without any indication of what happened. It would be useful to have an error message in this case.
> 
>     * There should also be some sort of instructions for successfully sending the reset email. Right now it's really unclear what to do once you click reset (unless you get an email notification, but I don't think we should rely on that)

I made one notification that will be displayed in both cases (one message to prevent info leakage)

**Review by t-whitney — APPROVED** [review]
- My only comment is that you could have separate messages for success and failure. Otherwise this looks good

**Pull request merged**